include/oneapi/dpl/internal/dynamic_selection_impl/inline_scheduler.h

```diff
@@ -0,0 +1,125 @@
+// -*- C++ -*-
+//===----------------------------------------------------------------------===//
+//
+// Copyright (C) Intel Corporation
+//
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+// This file incorporates work covered by the following copyright and permission
+// notice:
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+//
+//===----------------------------------------------------------------------===//
+
+#pragma once
+
+#include "oneapi/dpl/dynamic_selection"
+#include "oneapi/dpl/internal/dynamic_selection_impl/scoring_policy_defs.h"
+#include "oneapi/dpl/internal/dynamic_selection_impl/scheduler_defs.h"
+#include "oneapi/dpl/internal/dynamic_selection_impl/concurrent_queue.h"
+
+#include <ostream>
+#include <vector>
+#include <atomic>
+
+namespace oneapi {
+namespace dpl {
+namespace experimental {
+struct int_inline_scheduler_t {
```

Comment:
Looks like this class is only used in tests, and not needed for the implementation. Move it to the tests.